packages/db/src/network/test/mockProvider.ts

```diff
@@ -0,0 +1,56 @@
+import { logger } from "@truffle/db/logger";
```

Comment:
This is great! I wonder if we could/should mock the provider for where we test contractInstances? I think we actually use web3 right now...

Reply:
yeah that'd be nice follow-on work!